health-monitors/gpu-health-monitor/gpu_health_monitor/platform_connector/platform_connector.py

```
@@ -176,6 +177,9 @@ def health_event_occurred(
                                 platformconnector_pb2.Entity(entityType="GPU_UUID", entityValue=gpu_uuid)
                             )
 
+                        if pci_address and gpu_uuid:
+                            entities_impacted_supports_component_reset = true
```

Ruff (0.14.13)

Undefined name `true`

(F821)